lib/blpapi-wrapper.ts

```diff
@@ -287,7 +287,7 @@ export class Session extends events.EventEmitter {
         return Promise.all(_.map(subscriptionsToServices(subscriptions), (uri): Promise<void> => {
             return this.openService(uri);
         })).then((): void => {
-            log('Subscribing to: ' + JSON.stringify(subscriptions));
+            //log('Subscribing to: ' + JSON.stringify(subscriptions));
```

Comment:
For this, we probably need to have the `blpapi-wrapper.ts` export `Subscription` as a class instead of an interface to enforce only the properties it expects.